src/serviceFacades/apps.js

```diff
@@ -221,6 +366,13 @@ export {
     searchAppsInfiniteQuery,
     getAppDoc,
     saveAppDoc,
+    adminPatchApp,
```

Comment:
I still recommend not exporting this until it's need elsewhere.

Reply:
Just this end-point or all the end-points used in `adminUpdateApp` that are used elsewhere ?

Reply:
Ah, good point. I think all functions not used elsewhere should be removed from the export (like keeping methods `private` until they need to be `public`).